eng/tox/tox.ini

```diff
@@ -34,6 +34,8 @@ platform = linux: linux
            macos: darwin
            windows: win32
 passenv = *
+setenv =
```

Comment:
@lmazuel I'll check this in tomorrow. This is the sphinx-options addition I stated I'd add.